tests/Resources/Page/ODH/JupyterHub/AccessGroups.resource

```diff
@@ -0,0 +1,161 @@
+    IF    ':' not in $STANDARD_ADMINS_GROUP
+        Remove User From Group    username=${TEST_USER_2.USERNAME}    group_name=${STANDARD_ADMINS_GROUP}
+        Check User Is Not In A Group    username=${TEST_USER_2.USERNAME}   group_name=${STANDARD_ADMINS_GROUP}
+    END
+    IF    ':' not in $STANDARD_USERS_GROUP
+        Remove User From Group    username=${TEST_USER_3.USERNAME}    group_name=${STANDARD_USERS_GROUP}
+        Check User Is Not In A Group    username=${TEST_USER_3.USERNAME}   group_name=${STANDARD_USERS_GROUP}
+    END
+
+Add Test Users Back To RHODS Standard Groups
+    [Documentation]    Adds two tests users back to rhods-admins and rhods-users
+    IF    ':' not in $STANDARD_ADMINS_GROUP
+        Add User To Group    username=${TEST_USER_2.USERNAME}    group_name=${STANDARD_ADMINS_GROUP}
+        Check User Is In A Group    username=${TEST_USER_2.USERNAME}   group_name=${STANDARD_ADMINS_GROUP}
+    END
+    IF    ':' not in $STANDARD_USERS_GROUP
+        Add User To Group    username=${TEST_USER_3.USERNAME}    group_name=${STANDARD_USERS_GROUP}
+        Check User Is In A Group    username=${TEST_USER_3.USERNAME}   group_name=${STANDARD_USERS_GROUP}
+    END
+
+
+Remove Test Users From Custom Groups
+    [Documentation]    Removes two tests users from custom-admins and customer-users groups
+    Remove User From Group    username=${TEST_USER_2.USERNAME}    group_name=${CUSTOM_ADMINS_GROUP}
+    Remove User From Group    username=${TEST_USER_3.USERNAME}    group_name=${CUSTOM_USERS_GROUP}
+    Check User Is Not In A Group    username=${TEST_USER_2.USERNAME}   group_name=${CUSTOM_ADMINS_GROUP}
+    Check User Is Not In A Group    username=${TEST_USER_3.USERNAME}   group_name=${CUSTOM_USERS_GROUP}
+
+Check New Access Configuration Works As Expected
+    [Documentation]    Checks if the new access configuration (using two custom groups)
+    ...                works as expected in JH
+    Go To RHODS Dashboard
+    Launch JupyterHub From RHODS Dashboard Link
+    Handle Bad Gateway Page
+    Run Keyword And Continue On Failure   Login Verify Access Level  ${TEST_USER.USERNAME}
+    ...                                   ${TEST_USER.PASSWORD}    ${TEST_USER.AUTH_TYPE}    none
+    Capture Page Screenshot    perm_denied.png
+    Go To RHODS Dashboard
+    Launch JupyterHub From RHODS Dashboard Link
+    Run Keyword And Continue On Failure   Login Verify Access Level    ${TEST_USER_2.USERNAME}
+    ...                                   ${TEST_USER_2.PASSWORD}    ${TEST_USER_2.AUTH_TYPE}    admin
+    Capture Page Screenshot    perm_admin.png
+    Logout Via Button
+    Go To RHODS Dashboard
+    Launch JupyterHub From RHODS Dashboard Link
+    Run Keyword And Continue On Failure   Login Verify Access Level    ${TEST_USER_3.USERNAME}
+    ...                                   ${TEST_USER_3.PASSWORD}    ${TEST_USER_3.AUTH_TYPE}    user
+    Capture Page Screenshot    perm_user.png
+    Logout Via Button
```

Comment:
This is basically the same as `jupyterhub-user-access.robot`; I think this should not be a keyword and instead be a TC, better yet we can just update the existing TC to change the group configuration before starting.

Reply:
this is basically the core of the TC ods.293. I've made a kw to avoid  writing all of this code in the TC directly